RELEASE_NOTES.rst

```diff
@@ -175,8 +175,7 @@ are added for existing bears.
 
 New bears:
 -  ``AutoPrefixBear`` (Add vendor prefixes automatically in CSS)
--  ``ClangComplexityBear`` (Calculates cyclomatic complexity of each function
-    for C, C++ and other Clang supported languages.)
+-  ``ClangComplexityBear`` (Calculates cyclomatic complexity of each function for C, C++ and other Clang supported languages.)
```

Comment:
nono this is not what the issue wants i think, it simply wants you to allign the ``for`` with the first double backtick.

also the hyphens need to be at the same level.

what you did was move all text on the same line, we dont want that.